src/main/java/online/book/store/mapper/BookMapper.java

```diff
@@ -1,17 +1,39 @@
 package online.book.store.mapper;
 
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
 import online.book.store.config.MapperConfig;
 import online.book.store.dto.request.book.BookRequestDto;
+import online.book.store.dto.response.book.BookDtoWithoutCategoryIds;
 import online.book.store.dto.response.book.BookResponseDto;
 import online.book.store.model.Book;
+import online.book.store.model.Category;
+import org.mapstruct.AfterMapping;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
 
 @Mapper(config = MapperConfig.class)
 public interface BookMapper {
     BookResponseDto toDto(Book book);
 
     @Mapping(target = "id", ignore = true)
     @Mapping(target = "deleted", ignore = true)
+    @Mapping(target = "categories", ignore = true)
```

Comment:
"Mapping(target = "categories", ignore = true)" - I have only this one .

Reply:
Investigate is it possible to remove unnecessary annotations